services/order.py

```diff
@@ -0,0 +1,39 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+@transaction.atomic
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
```

Comment:
The `date` parameter should be a `datetime` object, not a string. This will ensure proper handling of date and time when setting the `created_at` field.